src/Transports.Subscriptions.Abstractions/Transports.Subscriptions.Abstractions.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
```

Comment:
Encoding?

Reply:
🤷‍♂️ I didn't purposefully make a change, or know how to change the encoding, or know what it was changed to, or what it was.